engine/src/main/java/org/pentaho/di/www/GetStatusServlet.java

```diff
@@ -294,7 +294,7 @@ public void doGet( HttpServletRequest request, HttpServletResponse response ) th
         out.println( "<table cellspacing=\"0\" cellpadding=\"0\" class=\"toolbar\" style=\"width: 100%; height: 26px; margin-bottom: 2px; border: 0;\">" );
         out.println( "<tbody><tr>" );
         out.println( "<td align=\"left\" style=\"vertical-align: middle; width: 100%\" id=\"trans-align\"></td>" );
-        out.println( "<td " + setupIconEnterLeaveJavascript( "pause" ) + " align=\"left\" style=\"vertical-align: middle;\"><div style=\"padding: 2px;\" onClick=\"resumeFunction( this )\" class=\"toolbar-button toolbar-button-disabled\" id=\"pause\"><img style=\"width: 22px; height: 22px\" src=\"" + prefix + "/images/run_option.svg\" title=\"" + run + "\"/></div></td>" );
+        out.println( "<td " + setupIconEnterLeaveJavascript( "pause" ) + " align=\"left\" style=\"vertical-align: middle;\"><div style=\"padding: 2px;\" onClick=\"resumeFunction( this )\" class=\"toolbar-button toolbar-button-disabled\" id=\"pause\"><img style=\"width: 22px; height: 22px\" src=\"" + prefix + "/images/run.svg\" title=\"" + run + "\"/></div></td>" );
```

Comment:
I can't find this change in the original PR which I guess is #6565.
Please confirm this is OK.

Reply:
That change is not included in the original code changes from PDI-17863 done by Dean DiRoma, but if you notice, the SVG filename was wrong under 8.2, causing a 404 for that image, and would differ from 8.3. Please advice @pentaho-lmartins if I should remove this change.

Reply:
![image](https://user-images.githubusercontent.com/4614970/58583761-b7766780-8222-11e9-8a44-c856612ca1e4.png)

![image](https://user-images.githubusercontent.com/4614970/58583731-a6c5f180-8222-11e9-8f62-b0567cf33bc8.png)

Reply:
@jpruiz114 , did you test the behaviour before using the newly built jar?
There was a PR [pentaho-kettle#6081](https://github.com/pentaho/pentaho-kettle/pull/6081) back in December 2018 for "[SP-4800] Backport for BACKLOG-26741 -- 8.2 Carte status page unable to acess Scaleable Vector Graphic (.svg) files" that may already fixed that.
Using 8.2.0.6-QAT-655, I have this:
![image](https://user-images.githubusercontent.com/6816863/58716336-b9762d00-83c0-11e9-8624-469d12d3f760.png)

So it seems safe to remove that extra code.

Reply:
@pentaho-lmartins , with the information that @jpruiz114 gathered (and left on [this comment](https://jira.pentaho.com/browse/SP-5069?focusedCommentId=370273&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-370273)), my opinion is to approve and merge this PR.
Do you agree?